Sage/carbon
# S1T-1840 - Removed css height of the scrollable list item to fix item text overlapping

# Description

Removed height css property of the scrollable list item to fix overlapping of the item text

# Related Issues

https://jira.sage.com/browse/S1T-1840

# Screenshots / Gifs

![image](https://user-images.githubusercontent.com/44167188/56037728-ed6d9400-5d27-11e9-860c-2e2cc2d97204.png)

# Testing Instructions

Make sure ledger accounts for corrections in the German locale don't overlap in the 'Make Correction' dialogue.
